packages/router/src/utils/view_transition.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/// <reference types="@types/dom-view-transitions" />
+
+import {afterNextRender, InjectionToken, Injector, NgZone} from '@angular/core';
+import {from, Observable, of} from 'rxjs';
+
+export const TRANSITION_HELPER =
```

Comment:
nit: I'm wondering if we should try to preserve the "view transition" as a term, so that it's easier to understand (without looking at its implementation or docs) what this token refers to.
```suggestion
export const VIEW_TRANSITION_HELPER =
```